.travis.yml

```diff
@@ -2,13 +2,15 @@
 sudo: false
 dist: xenial
 language: java
-jdk:
-  - openjdk8
-  - openjdk11
-install: true
-script:
-    # excluding wallet-template because of the unreliable JavaFX dependency
-  - gradle clean build -x :wallettemplate:compileJava
+install:
+  - sudo apt-get install -y protobuf-compiler
+
+matrix:
+  include:
+    - jdk: openjdk8
+      script: gradle clean build
+    - jdk: openjdk11
+      script: gradle -c settings-jdk11.gradle clean build
```

Comment:
Rather than using a separate settings-jdk11.gradle, did you try using the `-x` (exclude) switch?

Reply:
I did try that, the problem is the `wallettemplate` module is using a JDK11-requiring plugin and even with the `-x` switch the build still fails.

Reply:
Actually, I have an even better solution --- stay tuned.